django/river/common/analyzer/analyzer.py

```diff
@@ -127,36 +147,33 @@ def analyze_input_group(self, mapping_group, parent_attribute):
         input_group = InputGroup(id_=mapping_group["id"], attribute=parent_attribute)
         parent_attribute.add_input_group(input_group)
         for input_ in mapping_group["inputs"]:
-            if input_["staticValue"]:
-                input_group.add_static_input(input_["staticValue"])
+            if input_["static_value"]:
+                input_group.add_static_input(input_["static_value"])
 
-            elif input_["sqlValue"] and input_["sqlValue"]["table"]:
```

Comment:
are you sure we don't need the `and input_["sqlValue"]["table"]` anymore ?

Reply:
I think so. Now that we have constraints on the model, we can't have a column with an empty table, right?